src/golive/cdn.md

```diff
@@ -52,3 +52,61 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you might not want users to access your platform.sh origin directly. There are 2 ways to secure your origin.
+
+### 1 Basic HTTP Authentication
+
+#### Password protected
```

Comment:
We try to avoid level 4 headers.  Can this be restructured to not need that?

Reply:
we avoid them , because...?

Reply:
Too deep of a tree makes it harder for the reader to follow and keep track of where they are.

Also, off hand I'm not sure how far down we've styled the headers. :smile: